manta-trusted-setup/src/groth16/kzg.rs

```diff
@@ -253,6 +253,29 @@ where
                 .1,
         })
     }
+
+    /// Reinterpret a proof from a KZG ceremony as a proof
+    /// for a sub-ceremony.
+    #[inline]
+    pub fn cast_to_subceremony<D>(self) -> Proof<D>
+    where
+        D: Pairing<G1 = <C as Pairing>::G1, G2 = <C as Pairing>::G2>,
```

Comment:
This should probably work:

```suggestion
        D: Pairing<G1 = C::G1, G2 = C::G2>,
```